tests/test_digital.py

```diff
@@ -38,12 +38,12 @@ def test_noise_instance_model_validation() -> None:
 @pytest.mark.parametrize(
     "noise_config",
     [
-        [NoiseCategory.READOUT.INDEPENDENT],
-        [NoiseCategory.DIGITAL.BITFLIP],
-        [NoiseCategory.DIGITAL.BITFLIP, NoiseCategory.DIGITAL.PHASEFLIP],
+        [Noise.READOUT.INDEPENDENT],
+        [Noise.DIGITAL.BITFLIP],
+        [Noise.DIGITAL.BITFLIP, Noise.DIGITAL.PHASEFLIP],
     ],
 )
-def test_append(noise_config: list[NoiseCategory]) -> None:
+def test_append(noise_config: list[Noise]) -> None:
```

Comment:
A point that crossed my mind. Currently,

```python
Bitflip(0.1) | Bitflip(0.1)
```

Is allowed. Do we want to keep it ?